skbio/alignment/__init__.py

```diff
@@ -218,11 +218,14 @@
     local_pairwise_align_ssw,
 )
 from skbio.alignment._ssw_wrapper import StripedSmithWaterman, AlignmentStructure
+from skbio.alignment._path import AlignPath, PairAlignPath
```

Comment:
Make sure that the `toctree` above is documented. Otherwise these objects won't show up in the docs